### PR TITLE
Suggestion for Admin Trees: Find a new name

### LINKED ISSUE #2870: Suggestion for Admin Trees: Find a new name
The Admin Trees module is great but we should consider finding a more meaningful name for the users who do not get instantly what a navigation tree is and how it can help them to organize the menu and add custom links.

We could rename it by something like 'Admin links', 'Admin menu' or 'Custom menu'.

### PR DESCRIPTION
**Pull request by Skrypt**
Fixes #2870

### CODE REVIEW

**Comment by sebastienros** [comment]
What's missing?

**Comment by Skrypt** [comment]
Renaming classes that are named AdminTree to AdminMenu. If it's not needed then it can be merged.

**Comment by Skrypt** [comment]
Please test and review.

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/PlaceholderAdminNode.Fields.TreeEdit.cshtml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 @model PlaceholderAdminNodeViewModel
 
-<style asp-src="~/OrchardCore.AdminTrees/Styles/admin-trees.css"at="Head" depends-on="admin"></style>
-<style asp-src="~/OrchardCore.AdminTrees/Styles/admin-trees-icon-picker.css" at="Head" depends-on="admin"></style>
+<style asp-src="~/OrchardCore.AdminMenu/Styles/admin-menu.min.css" debug-src="~/OrchardCore.AdminMenu/Styles/admin-menu.css" at="Head" depends-on="admin"></style>
```
Comment (matiasmolleja) [inline src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/PlaceholderAdminNode.Fields.TreeEdit.cshtml]: I see that the corresponding views on the Contents and the Lists modules are still referencing the old asset files: admin-trees.css , admin-trees-icon-picker.css, admin-trees.js, admin-trees-icon-picker.js

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/TreeController.cs [hunk 2]
```diff
@@ -81,7 +79,7 @@ public async Task<IActionResult> List(AdminTreeListOptions options, PagerParamet
 
             var startIndex = pager.GetStartIndex();
             var pageSize = pager.PageSize;
-            IEnumerable<AdminTree> results = new List<AdminTree>();
+            IEnumerable<Models.AdminMenu> results = new List<Models.AdminMenu>();
 
             //todo: handle the case where there is a deserialization exception on some of the presets.
             // load at least the ones without error. Provide a way to delete the ones on error.
```
Comment (matiasmolleja) [inline src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/TreeController.cs]: "Error when retrieving the list of admin menus"

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Edit.cshtml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+@model ShapeViewModel<AdminMenuDeploymentStep>
+
+<h5>@T["Admin Trees"]</h5>
```
Comment (matiasmolleja) [inline src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Edit.cshtml]: Admin Menus

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Summary.cshtml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-@model ShapeViewModel<AdminTreesDeploymentStep>
+@model ShapeViewModel<AdminMenuDeploymentStep>
 
 <h5>@T["Admin Trees"]</h5>
```
Comment (matiasmolleja) [inline src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Summary.cshtml]: Menus

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Thumbnail.cshtml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-@model ShapeViewModel<AdminTreesDeploymentStep>
+@model ShapeViewModel<AdminMenuDeploymentStep>
 
 <h4 class="card-title">@T["Admin Trees"]</h4>
```
Comment (matiasmolleja) [inline src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Thumbnail.cshtml]: Menus

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/TreeController.cs (hunk 2 above)
Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/TreeController.cs]: You means with an Admin Menu that would come from the old module?

Comment (matiasmolleja) [reply src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/TreeController.cs]: No. Sorry. Not used to do reviews on github. :)
A few lines below there is an error message that needs to be changed from "Error when retrieving the list of admin trees" to "Error when retrieving the list of admin menus"

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/TreeController.cs]: ok I fixed it.

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/PlaceholderAdminNode.Fields.TreeEdit.cshtml (hunk 1 above)
Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/PlaceholderAdminNode.Fields.TreeEdit.cshtml]: done

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Edit.cshtml (hunk 3 above)
Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Edit.cshtml]: done

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Summary.cshtml (hunk 4 above)
Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Summary.cshtml]: done

File: src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Thumbnail.cshtml (hunk 5 above)
Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.AdminMenu/Views/Items/AdminTreesDeploymentStep.Fields.Thumbnail.cshtml]: done

**Comment by matiasmolleja** [comment]
I tried and it looks good.

**Pull request merged**